### ISSUE TITLE
Remote macOS: Can't solve "Mouse: not useable (blocked)"

### ISSUE THREAD

**Issue by jackfarzan** [RaiMan/SikuliX1#521]
I'm having an issue similar to issue #429, but my difference is I am accessing the machine remotely. My env is:

MacOS 11.3.1
Sikulix 2.0.5
Java version "11.0.10" 2021-01-19 LTS

I have tried removing and re-adding the Terminal permission in both Screen Recording and Accessibility, but it has not proven successful. What else can I try?

**Comment by RaiMan** [comment]
> but my difference is I am accessing the machine remotely

What exactly are you doing?

**Comment by jackfarzan** [comment]
@RaiMan would appreciate any additional insight on this, have been unable to make any progress.

**Comment by RaiMan** [comment]
Still not clear, what is running where.

**Comment by jackfarzan** [comment]
@RaiMan After some investigating, I was able to figure out the issue: 
1) Terminal needs to be able to access System Events in addition to Screen Recording and Accessibility
2) I had 2 Java versions on my machine, Java 11 which I mentioned earlier, and also Java 8 which I need for other purposes. Every time I ran Sikuli, I made sure it was using Java 11, however I kept getting the mouse permission error until I removed the Java 8 installation. 

How can I make sure that Sikuli uses the correct version of Java? I cannot permanently remove Java 8, both versions must coexist on this machine.
